### ISSUE TITLE
Write to backup log file when backup is finished

### ISSUE THREAD

**Issue by djmattyg007** [SparklingComet/AutoBackup#2]
IMO it would be more useful if backups were only logged to the backup log file once the backup is finished. It would be perfectly fine for both commencement and completion to be logged, but at the very least completion should be explicitly logged.

**Comment by SparklingComet** [comment]
Fair enough, I will add a setting to toggle the log message both before and after the completion.

**Comment by SparklingComet** [comment]
There has been a change of plans, commencement will not be logged anymore, only the end result (whether success or failure).

**Comment by djmattyg007** [comment]
How come? I think it would definitely be useful to have both. Knowing how long backups are taking is very valuable.

**Comment by SparklingComet** [comment]
I agree, but I’ll push commencement logs back to the release after the next because I wish to rewrite the logging system so it doesn’t make sense to implement something that will be rewritten in a couple weeks.

The next release will thus only contain the end log.
